Use the Xunit assert library instead of CoreCLRTestLibrary for most asserts (#61226)
dotnet/runtime: src/tests/Interop/ReadMe.md

Ancillary source assets for all tests should be located in `Interop/common` and can be easily added to all managed tests via the `Interop.settings.targets` file or native tests via `Interop.cmake`.

> A common pattern for testing is using the `Assert` utilities. This class is part of the `CoreCLRTestLibrary` which is included in all test projects by the `Interop.settings.targets` import. In order to use, add the following `using Xunit;` in the relevant test file.

Reviewer: ```suggestion
A common pattern for testing is using xUnit's `Assert` utilities. These utilities can be referenced via `CoreCLRTestLibrary` which is included in all test projects by `Directory.Build.targets` in this directory. In order to use, add the following `using Xunit;` in the relevant test file.
```

Reviewer 2: Fixed in latest commit.
